README.md

```diff
@@ -1,73 +1,23 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Project Name
 
-> Description the project.
+Portfolio: Mobile Version
 
+> Description the project.
+> Building a personal portfolio and it's responsive in different platforms.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
+-   Major languages - HTML and CSS are used to make this projects.
+-   Frameworks - There is no any frameworks used in this work.
+-   Technologies used
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+-   GitHub: [@githubhandle](https://github.com/Addisu87)
+-   Twitter: [@twitterhandle](https://twitter.com/AddisuTedla)
+-   LinkedIn: [LinkedIn](https://www.linkedin.com/in/addisu-tedla-8b4a10143/)
 
-## 📝 License
+# How to get started using your project locally
 
-This project is [MIT](./MIT.md) licensed.
+-   Click the Go-live option in your VS code and follow the link which is similar to this.
```

Comment:
- [ ] Please kindly follow this [template](https://github.com/microverseinc/readme-template) and fill out all the sub sections with information related to you and the project, as this will make your `readme file` more informative and professional.